README.md

```diff
@@ -1,126 +1,131 @@
 
 KoP implements the Kafka wire protocol on Pulsar by leveraging the existing components (such as topic discovery, the distributed log library - ManagedLedger, cursors and so on) that Pulsar already has.
 
-The following figure illustrates how is the Kafka-on-Pulsar protocol handler was implemented within Pulsar.
+The following figure illustrates how the Kafka-on-Pulsar protocol handler is implemented within Pulsar.
 
 ![](docs/kop-architecture.png)
 
-## Prerequisite
+# What's New in KoP 2.8.0
+The following new features are introduced in KoP 2.8.0.
 
-From 2.6.2.0 on, KoP `x.y.z.m` is based on Pulsar `x.y.z`, while `m` is the patch version number. See [here](integrations/README.md) for supported Kafka clients.
+- Offset management
+- Message encoding and decoding
+- OAuth 2.0 authentication
+- Metrics related to produce, fetch and request
 
-Before KoP 2.6.2.0, check the following requirements.
+## Enhancement 
+The following enhancement is added in KoP 2.8.0.
+- Support more clients for Kafka admin
+- Enable advertised listeners, so users can use Envoy Kafka filter directly
 
-| KoP version | Kafka client version | Pulsar version |
-| :---------- | :------------------- | :------------- |
-| [0.3.0](https://github.com/streamnative/kop/releases/tag/v0.3.0) | [Kafka client 1.x, 2.0~2.6](integrations/README.md) | [Pulsar 2.6.1](http://pulsar.apache.org/en/download/) |
-| [0.2.0](https://github.com/streamnative/kop/releases/tag/v0.2.0) | [Kafka client 2.0.0](https://kafka.apache.org/20/documentation.html) | [Pulsar 2.5.0](http://pulsar.apache.org/en/download/) |
-| [0.1.0](https://github.com/streamnative/kop/releases/tag/v0.1.0) | [Kafka client 2.0.0](https://kafka.apache.org/20/documentation.html) | [Pulsar 2.5.0](http://pulsar.apache.org/en/download/) |
+# Version compatibility
 
-The `pulsar-protocol-handler-kafka-${version}.nar` is the KoP binary file which can be loaded by Pulsar broker.
+Since Pulsar 2.6.2, KoP version changes with Pulsar version accordingly. The version of KoP `x.y.z.m` conforms to Pulsar `x.y.z`, while `m` is the patch version number. 
 
-## Enable KoP on your existing Apache Pulsar clusters
+| KoP version | Pulsar version |
+| :---------- | :------------- |
+| [2.8.0](https://github.com/streamnative/kop/releases/tag/v2.8.0) |Pulsar 2.8.0|
+| [2.7.1](https://github.com/streamnative/kop/releases/tag/v2.7.1) |Pulsar 2.7.1|
+| [2.7.0](https://github.com/streamnative/kop/releases/tag/v2.7.0) |Pulsar 2.7.0|
+| [2.6.3](https://github.com/streamnative/kop/releases/tag/v2.6.3) |Pulsar 2.6.3|
+| [2.6.2](https://github.com/streamnative/kop/releases/tag/v2.6.2) |Pulsar 2.6.2|
```

Comment:
There's no 2.8.0 currently. In addition, somehow there's no 2.6.3 and 2.7.1 releases. I think KoP 2.7.1.x is associated to Pulsar 2.7.1.

Reply:
If we don't have 2.7.1 and 2.6.3 KoP releases, I can remove them.
For the KoP 2.8.0, do we cut 2.8.0 branch based on the master before release? If yes, it's Ok to write it here since this readme is preparation for KoP 2.8.0.